src/Sulu/Bundle/SnippetBundle/Content/SnippetContent.php

```diff
@@ -26,9 +26,6 @@
 use Sulu\Component\Content\ContentTypeExportInterface;
 use Sulu\Component\Content\PreResolvableContentTypeInterface;
 
-/**
- * ContentType for Snippets.
- */
 class SnippetContent extends ComplexContentType implements ContentTypeExportInterface, PreResolvableContentTypeInterface
```

Comment:
Maybe its usefull to rename the `SnippetContent` Class to `SnippetSelection` for the uniformity. 
Think that should have only little effect on the end user, because the content type is still called "snippet_selection, but think thats cleaner =)

Reply:
yes that would be really nice to do, but unfortunately it is bc break. if somebody has extended that class, his application will break if we rename the class/file 😕